onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -129,7 +129,7 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   OrtMutex tensorrt_mu_;
   int device_id_;
   AllocatorPtr allocator_;
-  mutable int subgraph_id_ = 0;
+  mutable char model_path_[4096];
```

Comment:
Where does the 4096 come from? (put a comment explaining it if it's arbitrary?)

Reply:
That's the maximum possible path length. TRT parser is not willing to use wchar_t in Windows due to the code change effort, so we convert wchar_t to char at ORT side before send the path to the parser.